runhouse/rns/defaults.py

```diff
@@ -288,3 +288,15 @@ def data_collection_enabled(self) -> bool:
             return False
 
         return True
+
+    @property
+    def observability_enabled(self):
+        return not self.get("disable_observability", False)
+
+    def disable_observability(self):
```

Comment:
i think these should be in `rns_client`, like we have for other config values. which can check both env vars andd the config file for values

Reply:
but don't we want to to be similar to `Disabling Data Collection`? 
![image](https://github.com/user-attachments/assets/48ca1d70-7787-43fe-939f-22b08b51b73c)

if we move those methods to rns_client:
a. the user will not be able to change disable_observability easily via python (he'll need to change the config.yaml or the env var)
b. as mentioned before, it will not match the API of changing the data_collection flag.